kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/dsl/KoverVariantConfig.kt

```diff
@@ -46,6 +46,12 @@ public interface KoverCurrentProjectVariantsConfig: KoverVariantConfig {
      */
     public fun createVariant(variantName: String, block: Action<KoverVariantCreateConfig>)
 
+    /**
+     * Create custom report variant as a copy of the [originalVariantName] variant, however, with the new name [variantName].
+     * You cannot add information from other variants to the new one.
```

Comment:
Does it mean that I cannot configure it? If so, what is the use-case for copying variant without altering configuration?